requirements/quality.txt

```diff
@@ -4,125 +4,534 @@
-pytest-cov==2.7.1         # via -c requirements/constraints.txt, -r requirements/test.txt
-pytest-django==3.7.0      # via -c requirements/constraints.txt, -r requirements/test.txt
-pytest==4.5.0             # via -c requirements/constraints.txt, -r requirements/test.txt, pytest-cov, pytest-django
-python-dateutil==2.4.0    # via -c requirements/constraints.txt, -r requirements/test.txt, botocore, faker, freezegun, moto, xblock
-python-jose==3.2.0        # via -r requirements/test.txt, moto
-python-slugify==4.0.1     # via -r requirements/test.txt, code-annotations
-python-swiftclient==3.11.0  # via -c requirements/constraints.txt, -r requirements/test.txt
-pytz==2021.1              # via -r requirements/test.txt, django, edx-submissions, fs, moto, xblock
-pyyaml==5.4.1             # via -r requirements/test.txt, cfn-lint, code-annotations, edx-i18n-tools, moto, xblock
-requests==2.25.1          # via -r requirements/test.txt, docker, moto, python-swiftclient, responses
-responses==0.12.1         # via -r requirements/test.txt, moto
-rsa==4.7.1                # via -r requirements/test.txt, python-jose
-s3transfer==0.3.4         # via -r requirements/test.txt, boto3
-six==1.15.0               # via -r requirements/test.txt, astroid, aws-sam-translator, bleach, cfn-lint, django-simple-history, docker, ecdsa, edx-i18n-tools, edx-lint, freezegun, fs, fs-s3fs, html5lib, jsonschema, junit-xml, libsass, mock, moto, pytest, python-dateutil, python-jose, python-swiftclient, responses, tox, virtualenv, websocket-client, xblock
-sqlparse==0.4.1           # via -r requirements/test.txt, django
-sshpubkeys==3.3.1         # via -r requirements/test.txt, moto
-stevedore==3.3.0          # via -r requirements/test.txt, code-annotations, edx-django-utils
-testfixtures==6.17.1      # via -r requirements/test.txt
-text-unidecode==1.3       # via -r requirements/test.txt, faker, python-slugify
-toml==0.10.2              # via -r requirements/test.txt, pylint, tox
-tox==3.21.4               # via -r requirements/test.txt
-urllib3==1.26.3           # via -r requirements/test.txt, botocore, requests, responses
-virtualenv==20.4.2        # via -r requirements/test.txt, tox
-voluptuous==0.12.1        # via -c requirements/constraints.txt, -r requirements/test.txt
-wcwidth==0.2.5            # via -r requirements/test.txt, pytest
-web-fragments==1.0.0      # via -r requirements/test.txt, xblock
-webencodings==0.5.1       # via -r requirements/test.txt, bleach, html5lib
-webob==1.8.6              # via -r requirements/test.txt, xblock
-websocket-client==0.57.0  # via -r requirements/test.txt, docker
-werkzeug==1.0.1           # via -r requirements/test.txt, moto
-wrapt==1.11.2             # via -c requirements/constraints.txt, -r requirements/test.txt, astroid, aws-xray-sdk
-xblock-sdk==0.2.2         # via -r requirements/test.txt
-xblock==1.4.0             # via -r requirements/test.txt
-xmltodict==0.12.0         # via -r requirements/test.txt, moto
+appdirs==1.4.4
```

Comment:
Summary of changes:

astroid==2.4.2 to astroid==2.5.1
aws-sam-translator==1.34.0 to aws-sam-translator==1.35.0
boto3==1.17.8 to boto3==1.17.28
botocore==1.20.8 to botocore==1.20.28
cfn-lint==0.45.0 to cfn-lint==0.47.2
coverage==5.4 to coverage==5.5
cryptography==3.4.5 to cryptography==3.4.6
defusedxml==0.6.0 to defusedxml==0.7.1
django==2.2.18 to django==2.2.19
docker==4.4.2 to docker==4.4.4
edx-django-utils==3.13.0 to edx-django-utils==3.15.0
edx-lint==4.0.1 to edx-lint==4.1.0
faker==6.1.1 to faker==6.6.1
jsonpatch==1.28 to jsonpatch==1.32
jsonpointer==2.0 to jsonpointer==2.1
lazy-object-proxy==1.4.3 to lazy-object-proxy==1.5.2
newrelic==6.0.1.155 to newrelic==6.2.0.156
pycodestyle==2.6.0 to pycodestyle==2.7.0
pylint==2.6.2 to pylint==2.7.2
python-swiftclient==3.11.0 to python-swiftclient==3.11.1
rsa==4.7.1 to rsa==4.7.2
tox==3.21.4 to tox==3.23.0
urllib3==1.26.3 to urllib3==1.26.4
webob==1.8.6 to webob==1.8.7
websocket-client==0.57.0 to websocket-client==0.58.0